docs/ingress-gateway-asm-manifests/base/kustomization.yaml

```diff
@@ -19,4 +19,5 @@ resources:
 - deployment-service.yaml
 - gateway.yaml
 - namespace.yaml
+- pdb-v1.yaml
```

Comment:
**Nitpick (feel free to ignore):**
I understand that we're currently using the same **file names** from the [`anthos-service-mesh-packages` repo](https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/tree/main/samples/gateways/istio-egressgateway).
But I think:
* `pdb-v1.yaml` can be renamed to `pod-disruption-budget.yaml`.
* `autoscaling-v2beta1.yaml` can be renamed to `horizontal-pod-autoscaler.yaml`.

for consistency with existing file names (e.g., `gateway.yaml` and `namespace.yaml`) and for clarify.